src/charm.py

```diff
@@ -160,6 +160,8 @@ def ingress_config(self):
         tls_secret_name = self.model.config["tls_secret_name"]
         if tls_secret_name:
             ingress_config["tls-secret-name"] = tls_secret_name
+        ingress_config["owasp-modsecurity-crs"] = self.model.config["use_nginx_ingress_modsec"]
```

Comment:
I think we need to be a little careful here - I'm not sure we can turn this on without some custom exclusions. See https://code.launchpad.net/~sajoupa/charm-k8s-wordpress/+git/charm-k8s-wordpress/+merge/414554 for more information. This was preventing post requests from succeeding per Ubuntu RT#37351: "Ubuntu Fridge - unable to post (UWN 719)"